applications/IgaApplication/custom_utilities/anurbs.h

```diff
@@ -49,4 +49,40 @@ struct Zero<Kratos::array_1d<TScalar, TDimension>>
 } // namespace Internals
 } // namespace ANurbs
 
+namespace Kratos {
+
+template <int TDimension>
```

Comment:
@jcotela does this need approval of the committee?

Reply:
Options: give then a long name or put them into a namespace